### PR TITLE
Feature/lef 270 list organizations sites

### PR DESCRIPTION
**Pull request by roboli**
In order to keep things clean and working in the backend, I had to append the v2 to this module.

### CODE REVIEW

File: api-module-library/sharepoint/defaultConfig.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-    "name": "microsoft-sharepoint",
+    "name": "microsoft-sharepoint-v2",
```
Comment (seanspeaks) [inline api-module-library/sharepoint/defaultConfig.json:2]: @roboli I think we keep this as normal here and override it locally in the Frigg App. Anyone coming to this fresh shouldn't have to wonder what "v2" is all about. You'd override it in the "entities" directory, and either change the config variable or the GetName method.

File: api-module-library/sharepoint/jest-setup.js [hunk 2]
```diff
@@ -2,10 +2,10 @@ const dotenv= require('dotenv');
 const { globalSetup } = require('@friggframework/test-environment');
 
 const parsed = {
-    SHAREPOINT_SCOPE: 'sharepoint_scope_test',
-    SHAREPOINT_CLIENT_ID: 'sharepoint_client_id_test',
-    SHAREPOINT_CLIENT_SECRET: 'sharepoint_client_secret_test',
-    SHAREPOINT_REDIRECT_URI: 'http://redirect_uri_test'
+    SHAREPOINT_V2_SCOPE: 'sharepoint_scope_test',
```
Comment (seanspeaks) [inline api-module-library/sharepoint/jest-setup.js:5]: Keep these as is, override locally as needed

File: api-module-library/sharepoint/manager.js [hunk 3]
```diff
@@ -26,10 +26,10 @@ class Manager extends ModuleManager {
 
         // initializes the Api
         const sharepointParams = {
-            client_id: process.env.SHAREPOINT_CLIENT_ID,
-            client_secret: process.env.SHAREPOINT_CLIENT_SECRET,
-            redirect_uri: `${process.env.SHAREPOINT_REDIRECT_URI}/microsoft-sharepoint`,
-            scope: process.env.SHAREPOINT_SCOPE,
+            client_id: process.env.SHAREPOINT_V2_CLIENT_ID,
```
Comment (seanspeaks) [inline api-module-library/sharepoint/manager.js:29]: Keep these the same here, override locally as needed

**Review by leofmds — CHANGES_REQUESTED** [review]
@roboli We shouldn't change the name of this public integration due to an internal project. 
Instead, we should internalize what needs to be done to comply with the changed name (i.e. you get microsoft-sharepoint-v2 from the frontend, but for some reason you need to change it to microsoft-sharepoint in the backend).
The only thing I agree here is to change the SHAREPOINT_REDIRECT_URI to REDIRECT_URI. Leave it like that and rollback the rest.

**Pull request closed without merging**